core/spec.md

```diff
@@ -14,9 +14,12 @@ automation and tooling.
   - [Terminology](#terminology)
 - [Registry Attributes and APIs](#registry-attributes-and-apis)
   - [Attributes and Extensions](#attributes-and-extensions)
+    - [Common Attributes](#common-attributes)
+    - [`xid`](#xid)
   - [Registry APIs](#registry-apis)
     - [Registry Collections](#registry-collections)
     - [Entity Processing Rules](#entity-processing-rules)
+  - [`xid` Resolution API](#xid-resolution-api)
```

Comment:
any reason this isn't indented one more since it's still a "Registry API", right?

Reply:
I'm not sure it's a registry API. It may also be offered independently. When we work on the overall document structure we can revisit this.